test.js

```diff
@@ -15,3 +15,35 @@ test(function (t) {
     t.equal(string, 'foo,bar\nbar,baz\nbar1,baz1\n', 'outputs csv stream')
   })
 })
+
+test(function (t) {
```

Comment:
since you're adding several tests you should name each one, especially your custom tests. You may not need a message for your assertions, it's really the test that should be named (`test('name', (t) => {}`)